libs/components/datetime/src/lib/modules/datepicker/datepicker-config.service.ts

```diff
@@ -20,6 +20,12 @@ export class SkyDatepickerConfigService {
    */
   public minDate: Date | undefined;
 
+  /**
+   * The date to open the calendar to initially if there isn't a date currently selected.
```

Comment:
I think we're okay to keep it simple with: `The date to open the calendar to initially.` The word "initially" already communicates that it can be overridden with the datepicker's selected value after the fact.